src/main/java/com/bina/lrsim/simulator/Simulator.java

```diff
@@ -411,8 +438,7 @@ and assume it starts anywhere completely randomly (if sequencing starts
                 clrBegin = locus.getEnd0() - currentInsertLength;
               else
                 clrEnd = locus.getBegin0() + currentInsertLength;
-            }
-            if (isLastClr) {
+            } else if (isLastClr) {
```

Comment:
@yunfeiguo  this is a very very serious bug introduced in refactoring, see https://github.com/bioinform/longislnd/blob/master/src/main/java/com/bina/lrsim/simulator/Simulator.java#L130

we need to stop everything and find out when this happened